tests/test_conversion.py

```diff
@@ -171,12 +174,15 @@ def test_xml_tagvalue(self):
         self.write_tagvalue_file(doc, filename)
         self.parse_tagvalue_file(filename)
 
+
     def test_tagvalue_json(self):
         doc = self.parse_tagvalue_file(utils_test.get_test_loc('formats/SPDXTagExample.tag'))
         filename = get_temp_file('.json')
         self.write_json_file(doc, filename)
         self.parse_json_file(filename)
 
+    @unittest.skip("This fails because we can read/write the mandatory field snippet_byte_range so far only for "
+                   "json-, yaml- or xml-files, not for rdf-/tv-files. https://github.com/spdx/tools-python/issues/274")
```

Comment:
This comment and two more instances further down have to be adapted (they still claim tag/value doesn't work)

Reply:
done